.travis.yml

```diff
@@ -3,9 +3,21 @@ install:
   - npm install -g codecov
   - yarn --ignore-engines
 node_js:
-  - 8
+  - 10
 script:
-  - npm run lint
-  - npm run test
+  - yarn lint
+  - yarn test
   - codecov
-cache: yarn
\ No newline at end of file
+  - yarn build
+cache: yarn
+
+deploy:
+  provider: script
+  skip_cleanup: true
+  script: yarn publish
```

Comment:
Use custom script to handle monorepo deployment, it's using ocular under the hood.

Reply:
looks like you still need to test this out, @gnavvy make take a look of this.